### PR TITLE
Hi-Fi: Footer

### LINKED ISSUE #46: Hi-fi: Footer

### PR DESCRIPTION
**Pull request by isabellberzsenyi**
This PR contains code for the footer. Including refactoring of styling on the Contact Page (originally part of the footer was in this page), BoxLink and ImageBoxLink components. This PR also includes modifications to About, Contact and Story Inquiry page to add the footer and footer data.

### CODE REVIEW

**Review by josephmannis — CHANGES_REQUESTED** [review]
This looks good! Just to be sure, this PR is not responsible for these responsiveness issues?
If not, could you please make sure this issue is tracked (I'm running the site on my phone):

IMG_32828513DB57-1

Also, I'm seeing this error in the console. Could you resolve it before merging please?

image

**Comment by jean-zhang** [comment]
I'll be fixing the homepage in a separate PR!
https://github.com/Scout-NU/thealleyway-site/projects/1#card-59713986

File: components/footer/footerLink/FooterLink.style.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+const FooterLinkWrapper = styled(P).attrs({
+  as: 'a',
+})`
+  color: ${colors.WHITE};
+`;
+
+const FooterLinkContainer = styled.div`
+  margin: 0.25em;
+  &:hover {
```
Comment (jean-zhang) [inline components/footer/footerLink/FooterLink.style.js]: Do we want to touch base with the designers for what the hover state would be? I'm guessing underline or bold

Comment (isabellberzsenyi) [reply components/footer/footerLink/FooterLink.style.js]: Good pointing sent a message to jay about this

**Review by jzhzhang66 — CHANGES_REQUESTED** [review]
image
From around screen width 324 - 354 ish the email is overlapping so maybe move the ig, facebook, twitter, etc. stuff below the email starting at a higher screen width 
^^ this would also be good because there's no space on the right if its all in one line like in this photo

File: components/imageBoxLink/ImageBoxLink.styles.js [hunk 2]
```diff
@@ -1,40 +1,35 @@
...
-  @media ${min.desktop} {
-    margin: 1em 0 0 40em;
+  @media ${max.tabletSm} {
+    margin: 2em auto 1em auto;
   }
 `;
 
 const BottomImage = styled.img`
-  opacity: 10%;
+  opacity: 30%;
```
Comment (jzhzhang66) [inline components/imageBoxLink/ImageBoxLink.styles.js]: I believe this opacity should be 10% based off of hi-fis?

**Pull request merged**